src/Platform/Microsoft.Testing.Platform/Logging/FileLoggerProvider.cs

```diff
@@ -5,7 +5,9 @@
 
 namespace Microsoft.Testing.Platform.Logging;
 
-internal sealed class FileLoggerProvider : ILoggerProvider, IDisposable
+#pragma warning disable CA1852
```

Comment:
You probably crafted this disable manually, could you add the `// rule message` part either manually or let the lightbulb add the suppression.

Reply:
Indeed, I've manually added this suppression. It's strange because nothing is wrong and VS doesn't warn of it as a consequence, but when I build in CLI I get an error. The error I get is because the analyzer thinks the class can be sealed, however it cannot because it has a virtual member. Will manually add the rule description too.

Reply:
Removed the warning suppression.